test/integration/master/master_test.go

```diff
@@ -73,6 +73,16 @@ func TestExtensionsPrefix(t *testing.T) {
 	testPrefix(t, "/apis/extensions/")
 }
 
+func TestKubernetesService(t *testing.T) {
+	config := framework.NewMasterConfig()
+	_, _, closeFn := framework.RunAMaster(config)
```

Comment:
Can we get a `RunARealMaster` method to actually start a real server or is that going to be a followup?